internal/users/service/user_service.go

```diff
@@ -163,14 +171,11 @@ func (s *userServiceImpl) Logout(ctx context.Context, id uuid.UUID) error {
 		return err
 	}
 
-	apiKeys, err := s.apiKeyRepo.GetForUser(ctx, id)
+	tx, err := s.conn.StartTransaction(ctx)
```

Comment:
Instead of getting the API keys and then deleting them we now directly delete them for a user.